src/js/api/media/Manager.js

```diff
@@ -65,8 +65,6 @@ const Manager = function(container, browserInfo){
     that.destroy = () =>{
         $container.removeChild();
         $container = null;
-        videoElement.src = null;
```

Comment:
These codes were added due to the following issue.
I think We need to test to see if this issue doesn't occur.

https://github.com/AirenSoft/OvenPlayer/issues/361
https://github.com/AirenSoft/OvenPlayer/commit/00581a88393c9dbd44a84189c0a84b3ee52d354f

Reply:
> These codes were added due to the following issue. I think We need to test to see if this issue doesn't occur.
> 
> #361 [00581a8](https://github.com/AirenSoft/OvenPlayer/commit/00581a88393c9dbd44a84189c0a84b3ee52d354f)

@SangwonOh i've tested on safari, srcObject is required and does not gave us fetch /null

I've returned it